cmd/root.go

```diff
@@ -89,6 +97,17 @@ func init() {
 	RootCmd.PersistentFlags().StringVarP(&cfgDir, "config-dir", "D", "./cfg/", "config directory")
 	RootCmd.PersistentFlags().StringVar(&kubeVersion, "version", "", "Manually specify Kubernetes version, automatically detected if unset")
 
+	RootCmd.PersistentFlags().StringVar(&nodeFile,
```

Comment:
Persuade me otherwise, but I'm not convinced users need to be able to specify node or master YAML files individually. Can't they achieve the same thing by specifying a config directory that has the master and/or node test files they want? 

If there is a reason to add this, is the default (at least as it shows up in the help text) correct? I don't think it's really `/node.yaml` in the root directory, it's `<cfg>/<version>/node.yaml`

Reply:
You're right, I did this because I wanted to keep the same parameters as the `node` and `master` command. Are those also incorrect ? I think the original intention was to be able to provide a global override for those files, but it's always used inside the config dir or may be just allow to use a different name ?

```golang
path = filepath.Join(cfgDir, fileVersion)
...
def := filepath.Join(path, file)
```
I definitively agree with you config-dir is enough. 

If we remove these two flags we'll need to add a default global value for the root command:

```golang
var (
	envVarsPrefix      = "KUBE_BENCH"
	defaultKubeVersion = "1.6"
	kubeVersion        string
	cfgFile            string
	cfgDir             string
	jsonFmt            bool
	pgSQL              bool
	checkList          string
	groupList          string
	masterFile         = "master.yaml"
	nodeFile           = "node.yaml"
```

So either I removed it or we rework the description to fit an alternative file name flag.

WDYT ?

Reply:
Ah, yes I see what you mean, I didn't notice these already exist in master.go and node.go! For the purpose of this PR, let's just have the default global values. Let's leave them as-is for the master & node commands - I doubt they are used, and I think they're wrong, but we should deal with retiring them (and the federated version too) as a separate issue.)

Reply:
will do !